eng/pipelines/source-build-license-scan.yml

```diff
@@ -0,0 +1,124 @@
+schedules:
+- cron: "0 7 * * 1-5"
+  displayName: Run on weekdays at 7am UTC
+  branches:
+    include:
+    - main
+    - release/*
+
+pr: none
+trigger: none
+
+variables:
+  installerRoot: '$(Build.SourcesDirectory)/src/installer'
+
+jobs:
+- job: Setup
+  pool:
+    name: NetCore1ESPool-Svc-Internal
+    demands: ImageOverride -equals 1es-ubuntu-2004
+  steps:
+  - script: |
+      vmrSrcDir="$(Build.SourcesDirectory)/src"
+      
+      # Builds an Azure DevOps matrix definition. Each entry in the matrix is a path,
+      # allowing a job to be run for each src repo.
+      matrix=""
+      for dir in $vmrSrcDir/*/
+      do
+        if [ ! -z "$matrix" ]; then
+          matrix="$matrix,"
+        fi
+        repoName=$(basename $dir)
+        #matrix="$matrix{ \"$repoName\": { \"repoPath\": \"$dir\" } }"
+        matrix="$matrix \"$repoName\": { \"repoPath\": \"$dir\" }"
+      done
+
+      matrix="{ $matrix }"
+
+      echo "##vso[task.setvariable variable=matrix;isOutput=true]$matrix"
+    name: GetMatrix
+    displayName: Get Matrix
+
+- job: LicenseScan
+  dependsOn: Setup
+  pool:
+    name: NetCore1ESPool-Svc-Internal
+    demands: ImageOverride -equals 1es-ubuntu-2004
+  timeoutInMinutes: 420
+  strategy:
+    matrix: $[ dependencies.Setup.outputs['GetMatrix.matrix'] ]
+  steps:
+
+  - task: UseDotNet@2
```

Comment:
IMO prep should be utilized here to only install the SDK.  We should be installing the same SDK version defined by the VMR.  Having a different version here is adding another version that would need to be kept up-to-date.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/eced090485cfc07ce45abe717002fdd5ef71d25f